### ISSUE TITLE
Bedwars spawn bug

### ISSUE THREAD

**Issue by DevilMayank** [MCIndiaNetwork/MCI-Server#44]
### ✅ Server

- [ ] Survival
- [ ] Oneblock
- [ ] Skyblock
- [ ] Minigames
- [X] Bedwars

### 📋 Description

When a game ended and when we spawned back in the lobby 3 players who were the last alives [(shreyaa_ , zeltrax both of one team yellow) (kingshivam of team red) their names in tab and normally above their head showed like inside a game.
![image](https://user-images.githubusercontent.com/99308377/156428577-b5608a34-8559-468a-8463-dee0f5a108da.png)

### 📑 Reproduction Steps (Optional)

1. Go to a game end it 
2. Spawn back in lobby
3. Then press the tab or the last one alives you will see it

### 💡 Expected Behavior

To not the previous games team colors in the lobby

### 🎮 Minecraft Version

1.8.9

### 🎮 Minecraft Client Mod

Vanilla

### 🎮 Operating system 

Windows

### 💡 Extra Client Mods (Optional)

NA

**Comment by SpikeyHelmet** [comment]
Yes this seems to be a problem only occuring with 1.8 clients. Plugin devs are working on a fix.
